src/pinning.js

```diff
@@ -327,6 +321,15 @@ class Pinning {
     })
   }
 
+  async _pinDID (did) {
```

Comment:
As far as I can tell, this doesn't actually pin the DID - is this a legacy function that was kept in if we want to re-enable it, or should we use the `{ pin: true}` option with the resolver?

Reply:
Yes, we should definitely use the pin: true option. I'm quite surprised that we didn't do this already!